test/test.js

```diff
@@ -9,6 +9,15 @@ var hashFromString  = helpers.hashFromString;
 var mergeHash       = helpers.mergeHash;
 var replaceEmpty    = helpers.replaceEmpty;
 
+var emptyTranslationWithPaths = {
+    'msgstr': '',
+    'paths': ['']
+};
+
+// use with emptyTranslationWithPaths to prevent mutating the original
+function simpleDeepCopy(obj) {
+    return JSON.parse(JSON.stringify(obj));
+}
```

Comment:
It looks like every single place use the same argument. Why not making a `emptyTranslationWithPaths()` method instead? It would simplify the code a little bit.